frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditTable/EditTable.tsx

```diff
@@ -60,7 +60,7 @@ const transformTableFieldToEditForm = (
 
   return {
     ...pick(field, EDIT_TABLE_FIELD_KEYS),
```

Comment:
should we update the `EDIT_TABLE_FIELD_KEYS` constant instead to include the `addMoreRows` prop?

Reply:
oops yes, `EDIT_TABLE_FIELD_KEYS` already actually contains `addMoreRows`. Don't code at 1:30am, people

fixed in [7c7c3b4](https://github.com/opengovsg/FormSG/pull/4945/commits/7c7c3b40971f048c7a4810f1476589c85da8619e)